conan/tools/apple/xcodedeps.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, conanfile):
         self._conanfile = conanfile
         self.configuration = conanfile.settings.get_safe("build_type")
         self.architecture = conanfile.settings.get_safe("arch")
+
+        if self.architecture == "armv8":
+            self.architecture = "arm64" # Xcode uses arm64 instead of armv8
```

Comment:
We already have a conversion method, please check: `conan.tools.apple.apple.to_apple_arch`

Reply:
Oh thanks for that, will change it